config.go

```diff
@@ -1155,6 +1148,22 @@ func parseRPCParams(cConfig *chainConfig, nodeConfig interface{}, net chainCode,
 			confFile = "litecoin"
 		}
 
+		// If all of RPCUser, RPCPass, ZMQBlockHost, and ZMQTxHost are
+		// set, we assume those parameters are good to use.
+		if conf.ZMQPubRawBlock != "" && conf.ZMQPubRawTx != "" {
+			if conf.RPCUser != "" && conf.RPCPass != "" {
+				return nil
+			}
+			// Try to get RPCUser and RPCPass from an auth cookie
+			rpcUser, rpcPass, err :=
+				getBitcoindRPCCredentialsFromCookie(confDir)
+			if err == nil {
+				conf.RPCUser, conf.RPCPass = rpcUser, rpcPass
+				return nil
+			}
+			fmt.Println(err)
```

Comment:
Why print instead of return error here?

Reply:
@halseth because we don't have to stop and return here, the print is purely informational (otherwise one can't know that the cookie e.g. contains garbage, or the file wasn't found).

Reply:
In that case, would suggest adding some additional context to it like so:

`Unable to retrieve RPC credentials from cookie (path=/path/to/cookie), falling back to rpcuser/rpcpassword`

Reply:
Done.